lib/rubocop/cop/style/single_line_methods.rb

```diff
@@ -32,6 +35,7 @@ class SingleLineMethods < Base
 
         def on_def(node)
           return unless node.single_line?
+          return if node.endless?
```

Comment:
Shouldn't `#single_line?` return true on this case as well?

Reply:
My bad. I just realized the cop treats specially empty definitions and now endless definition. I guess we need to improve its description a bit.

Reply:
@bbatsov just realized my PR description was a bit wonky so I've fixed that.

`single_line?` likely returns true for an endless method (since they generally should be on a single line I believe), however, a single line method as far as this cop is concerned is one that is of the form `def x; something end`, and the correction code was failing because `node.loc.end` doesn't exist for an endless method.

Reply:
Yeah, I figured this out with a bit of a delay. 😆